api/modules/Catalog/Catalog.Application/Products/Delete/v1/DeleteProductResponse.cs

```diff
@@ -0,0 +1,2 @@
+﻿namespace FSH.WebApi.Catalog.Application.Products.Delete.v1;
+public sealed record DeleteProductResponse(string responseMessage);
```

Comment:
ideally, a REST delete endpoint should return a status code of 204 No Content.

Reply:
done